docs/cfp.md

```diff
@@ -7,66 +7,150 @@ hide:
 
-The program committee will discuss the submissions and their reviews and select the program over a video conference meeting. Submissions will be assessed as-is, with no expectation of substantive revision after peer review, though some submissions may be accepted conditional on specified changes (“shepherded”).
+### Submission procedure
 
-//Submit your manuscript using the [SC23 web system](https://submissions.supercomputing.org/?page=Submit&id=SC23WorkshopCANOPIEHPCSubmission&site=sc23).
+Accepted manuscripts will be published in IEEE Computer Society.  Manuscripts
+should use the IEEE Transactions format; templates are available for Microsoft
+Word and LaTeX (though note that the IEEEtran class comes with most LaTeX
+installations). Manuscripts must be at least 6 pages in the IEEE Transactions
+format, including figures, tables, and references. There is no maximum, but
+writing should be concise, and manuscript length must be commensurate with
+contribution size. 
+
+We enthusiastically welcome original, high-quality submissions within the scope
+above. These may describe complete studies; work-in-progress research; position
+papers on controversial, emerging, or hot topics; state of the practice; or any
+other manuscript the authors believe should be included in the CANOPIE-HPC
+program. We encourage submissions from academia, industry, government, and/or
+any other type of institution.  Each manuscript will be assessed using peer
+review by program committee members (or outside reviewers, if needed) on the
+basis of scientific validity, impact to the field, reproducibility,
+inclusivity, and opportunity for useful and lively discussion at the workshop.
+Review will be single-masked; i.e., reviewers will know authors’ identities,
+but not vice versa. Authors should not anonymize their manuscripts. Each
+manuscript submission will be checked using anti-plagiarism software and can
+expect to have at least 3 reviews.
+
+In conjunction with the SC Reproducibility Initiative, submissions should be as
+transparent as possible regarding all methods. Submissions are also expected to
+provide reproducibility artifacts such as reproduction instructions, source
+code, build recipes, and/or container images. These should be included as part
+of the submission in the form of 
+[Artifact Description/Artifact Evaluation appendices](https://sc23.supercomputing.org/program/papers/reproducibility-appendices-badges/),
+which adhere to an established format adopted by SC23.
+
+The program committee will discuss the submissions and their reviews and select
+the program over a video conference meeting. Submissions will be assessed
+as-is, with no expectation of substantive revision after peer review, though
+some submissions may be accepted conditional on specified changes
+(“shepherded”).
+
+Submit your manuscript using the [SC23 web system](https://submissions.supercomputing.org/?page=Submit&id=SC23WorkshopCANOPIEHPCSubmission&site=sc23).
```

Comment:
The link sends directly into the paper submission track.
For clarity, we should provide separate links for papers and lightning talks (once we get the latter page enabled, of course), e.g. something like this

Submit your work using the SC23 web system: [Papers] | [Lightning talks]

Reply:
Agree, I'll add this when we hear back.